functions.php

```diff
@@ -14,4 +14,17 @@ function swwrc_child_enqueue_scripts() {
 	wp_enqueue_style( 'ascent-style', get_stylesheet_directory_uri() . '/scss/swwrc.css' );
 	wp_enqueue_script( 'swwrc-videobg', get_stylesheet_directory_uri() . '/js/jQuery.videobg.js', array( 'jquery' ), spine_get_script_version(), true );
 	wp_enqueue_script( 'swwrc-custom', get_stylesheet_directory_uri() . '/custom.js', array( 'jquery' ), spine_get_script_version(), true );
-}
\ No newline at end of file
+}
+
+add_filter( 'pre_get_posts', 'projects_104b' );
+/**
+ * Query the `wsuwp_uc_project` post type for the 104b category archive.
+ */
+function projects_104b( $query ) {
+  if ( is_category( '104b' ) ) {
```

Comment:
We should have an additional check here for a true `$query->is_main_query()` so that we are only changing the main query of the page load.